### PR TITLE
Ap 3265 Bank Statement blue result panel

### PR DESCRIPTION
**Pull request by willc-work**
## What
[Link to story](https://dsdmoj.atlassian.net/browse/AP-3265)
Add new result panel
Remove call to CFE for these cases (handle this in limited way)
skip the capital_assessment_pages and go to a new page to display a result panel without any cfe call.

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: config/locales/en/providers.yml [hunk 1]
```diff
@@ -478,6 +479,15 @@ en:
         uploaded: Uploaded
         title: Files added
         no_files: Files uploaded will appear here.
+    no_eligibility_assessments:
+      show:
+        page_title: A caseworker needs to review your application
+        no_cfe_assessment: A caseworker needs to review your application
+        heading: A caseworker needs to review your application
+        cannot_calculate: We cannot calculate your client's financial eligibility yet because you uploaded bank statements
```
Comment (RoseSAK) [inline config/locales/en/providers.yml:487]: this is the most minor nitpick ever but should there be a full stop here?

File: config/locales/en/shared.yml [hunk 2]
```diff
@@ -75,6 +75,11 @@ en:
       apply_ref: 'Apply service case reference:'
       ccms_ref_html: "<abbr title='Client and Cost Management System'>CCMS</abbr> case reference:"
     assessment_results:
+      no_cfe_result:
+        heading: A caseworker needs to review your application
+        cannot_calculate: We cannot calculate your client's financial eligibility yet because you uploaded bank statements
```
Comment (RoseSAK) [inline config/locales/en/shared.yml:80]: same here^^

**Review by RoseSAK — APPROVED** [review]
LGTM 👍

File: config/locales/en/shared.yml (hunk 2 above)
Comment (willc-work) [reply config/locales/en/shared.yml:80]: will make this change on the next ticket

File: config/locales/en/providers.yml (hunk 1 above)
Comment (willc-work) [reply config/locales/en/providers.yml:487]: will make this change on the next ticket

**Pull request merged**